src/apps/beefy/api.ts

```diff
@@ -6,6 +6,7 @@ export type BeefyVault = {
   id: string
   name: string
   type: string // cowcentrated, gov
+  subType: string
```

Comment:
is there a set of values that this can contain? Maybe we can use specific types for `type` and `subType`.

Reply:
It appears there are, they are not all located in one place throughout the code in beefy. What would be the value in adding them in right now though?